dealroom_firestore_connector/__init__.py

```diff
@@ -58,8 +58,27 @@ def get(doc_ref, *args, **kwargs):
             # log error
             __log_exception(3, doc_ref, identifier, True)
             return ERROR
+        
 
-
+def _parse_payload(obj, key, replace_value):
+    """Returns the passed payload with the values for the specified key
+       replaced by the replaced_value (nested)
+    Args:
+        obj (dict): payload to parse
+        key (str): key to look for
+        replace_value (object): value to add
+    """
+    for k, v in obj.items():
+      if isinstance(v, dict):
+        obj[k] = _parse_payload(v, key, replace_value)
+      if isinstance(v, list):  
+        obj[k] = [_parse_payload(i) for i in v]
+      if key in obj:
+        obj[key] = replace_value
+    return obj
+    
```

Comment:
It searches recursively for the key passed as parameter, even if the data contains dictionaries or lists